tests/test_stack.py

```diff
@@ -0,0 +1,133 @@
+from stack.is_consecutive import first_is_consecutive, second_is_consecutive
+from stack.is_sorted import is_sorted
+from stack.remove_min import remove_min
+from stack.stutter import first_stutter, second_stutter
+from stack.switch_pairs import first_switch_pairs, second_switch_pairs
+from stack.valid_parenthesis import is_valid
+from stack.simplify_path import simplify_path
+from stack.stack import Array_stack, Linkedlist_stack
+
+import unittest
+class TestSuite(unittest.TestCase):
+    def test_is_consecutive(self):
+        self.assertTrue(first_is_consecutive([3, 4, 5, 6, 7]))
+        self.assertFalse(first_is_consecutive([3, 4, 6, 7]))
+        self.assertFalse(first_is_consecutive([3, 2, 1]))
+
+        self.assertTrue(second_is_consecutive([3, 4, 5, 6, 7]))
+        self.assertFalse(second_is_consecutive([3, 4, 6, 7]))
+        self.assertFalse(second_is_consecutive([3, 2, 1]))
+
+    def test_is_sorted(self):
+        # Test case: bottom [6, 3, 5, 1, 2, 4] top
+        self.assertFalse(is_sorted([6, 3, 5, 1, 2, 4]))
+        self.assertTrue(is_sorted([1, 2, 3, 4, 5, 6]))
+
+    def test_remove_min(self):
+        # Test case: bottom [2, 8, 3, -6, 7, 3] top
+        self.assertEqual([2, 8, 3, 7, 3], remove_min([2, 8, 3, -6, 7, 3]))
+        # Test case: 2 smallest value [2, 8, 3, 7, 3]
+        self.assertEqual([4, 8, 7], remove_min([4, 8, 3, 7, 3]))
+
+    def test_stutter(self):
+        # Test case: bottom [3, 7, 1, 14, 9] top
+        self.assertEqual([3, 3, 7, 7, 1, 1, 14, 14, 9, 9],
+                         first_stutter([3, 7, 1, 14, 9]))
+        self.assertEqual([3, 3, 7, 7, 1, 1, 14, 14, 9, 9],
+                         second_stutter([3, 7, 1, 14, 9]))
+
+    def test_switch_pairs(self):
+        # Test case: even number of values in stack
+        # bottom [3, 8, 17, 9, 1, 10] top
+        self.assertEqual([8, 3, 9, 17, 10, 1],
+                         first_switch_pairs([3, 8, 17, 9, 1, 10]))
+        self.assertEqual([8, 3, 9, 17, 10, 1],
+                         second_switch_pairs([3, 8, 17, 9, 1, 10]))
+        # Test case: odd number of values in stack
+        # bottom [3, 8, 17, 9, 1] top
+        self.assertEqual([8, 3, 9, 17, 1],
+                         first_switch_pairs([3, 8, 17, 9, 1]))
+        self.assertEqual([8, 3, 9, 17, 1],
+                         second_switch_pairs([3, 8, 17, 9, 1]))
+
+    def test_is_valid_parenthesis(self):
+
+        self.assertTrue(is_valid("[]"))
+        self.assertTrue(is_valid("[]()[]"))
+        self.assertFalse(is_valid("[[[]]"))
+        self.assertTrue(is_valid("{([])}"))
+        self.assertFalse(is_valid("(}"))
+
+    def test_simplify_path(self):
+        p = '/my/name/is/..//keon'
+        self.assertEqual('/my/name/keon', simplify_path(p))
+
+
+class Test_Stack(unittest.TestCase):
```

Comment:
Why another class for it?

Reply:
Because I think stack.py is a class with many methods. Other files is just function. I think it is more clear :)

Reply:
@danghai Please use ```TitleCase``` for class names.